src/main/java/edu/metrostate/ics499/team2/controllers/CompoundController.java

```diff
@@ -0,0 +1,66 @@
+package edu.metrostate.ics499.team2.controllers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.metrostate.ics499.team2.model.ElementToValidate;
+import edu.metrostate.ics499.team2.services.ElementService;
+
+@RestController
+@RequestMapping("/compound")
+public class CompoundController {
+	private final Logger LOG = LoggerFactory.getLogger(getClass());
+	
+	private RestTemplate restTemplate = new RestTemplate();
+	
+	@GetMapping(value = "validate")
+	public Map<String, ?> process(@RequestBody  ElementToValidate payload) {
+		String prefix = "https://pubchem.ncbi.nlm.nih.gov/rest/pug/compound/fastformula/";
```

Comment:
If we did go this route of constructing PubChem URLs on our backend, there is a little more work to do.